src/os/bluestore/KernelDevice.h

```diff
@@ -23,6 +23,7 @@
 #include "include/utime.h"
 
 #include "ceph_aio.h"
+#include "ceph_io_uring.h"
```

Comment:
no need to include this header here. as it's not part of the interface exposed by this header file.